### PR TITLE
mgr/cephadm: Add CEPHADM_PAUSED to test_cli

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/cephadm/module.py [hunk 1]
```diff
@@ -462,8 +462,14 @@ def _serve_sleep(self):
...
 
-    def serve(self):
-        # type: () -> None
+    def serve(self) -> None:
+        """
+        The main loop of cephadm.
+
+        command handler typically change the declarative state
+        of cephadm and then here, we're applying the state to the
+        real world.
```
Comment (mgfritch) [inline src/pybind/mgr/cephadm/module.py:471]:
```suggestion
        A command handler will typically change the declarative state
        of cephadm. This loop will then attempt to apply this new state. 
```

?

**Comment by sebastian-philipp** [comment]
green: https://pulpito.ceph.com/swagner-2020-07-01_10:16:24-rados:cephadm-wip-swagner3-testing-2020-07-01-1013-distro-basic-smithi/

still lots of failures:

* https://tracker.ceph.com/issues/46300
* 503 from the fedora RPM mirrior
* https://github.com/ceph/ceph/pull/35846

**Comment by sebastian-philipp** [comment]
https://pulpito.ceph.com/swagner-2020-07-15_11:08:12-rados:cephadm-wip-swagner3-testing-2020-07-15-1008-distro-basic-smithi/

**Comment by sebastian-philipp** [comment]
```
flake8 create: /home/jenkins-build/build/workspace/ceph-pull-requests/qa/.tox/flake8
flake8 installdeps: flake8
WARNING: Discarding $PYTHONPATH from environment, to override specify PYTHONPATH in 'passenv' in your configuration.
flake8 installed: -f /home/jenkins-build/build/workspace/ceph-pull-requests/qa/wheelhouse,flake8==3.8.3,importlib-metadata==1.7.0,mccabe==0.6.1,pycodestyle==2.6.0,pyflakes==2.2.0,zipp==3.1.0
flake8 run-test-pre: PYTHONHASHSEED='1607252996'
flake8 run-test: commands[0] | flake8 --select=F,E9 --exclude=venv,.tox
./tasks/cephadm_cases/test_cli.py:43:9: F841 local variable 'out' is assigned to but never used

**Comment by sebastian-philipp** [comment]
jenkins test dashboard backend

**Comment by sebastian-philipp** [comment]
https://pulpito.ceph.com/swagner-2020-07-20_10:53:40-rados:cephadm-wip-swagner-testing-2020-07-20-1103-distro-basic-smithi/

**Comment by sebastian-philipp** [comment]
jenkins test dashboard backend

**Pull request merged**